doc/source/reference/c-api.array.rst

```diff
@@ -122,9 +122,11 @@ sub-types).
 
 .. c:function:: PyObject *PyArray_GETITEM(PyArrayObject* arr, void* itemptr)
 
-    Get a Python object from the ndarray, *arr*, at the location
-    pointed to by itemptr. Return ``NULL`` on failure.
-
+    Return (builtin) Python types and get a Python object from the ndarray, *arr*, 
```

Comment:
How about `Get a python object of a builtin type from the array`?

Reply:
Sounds good